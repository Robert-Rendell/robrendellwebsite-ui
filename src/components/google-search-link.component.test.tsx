import React from "react";
import { render } from "@testing-library/react";
import { GoogleSearchLink } from "./google-search-link.component";

describe("GoogleSearchLink", () => {
  test("should match snapshot", () => {
    const { asFragment } = render(<GoogleSearchLink query="roberto"/>);
    expect(asFragment()).toMatchSnapshot();
  });
});