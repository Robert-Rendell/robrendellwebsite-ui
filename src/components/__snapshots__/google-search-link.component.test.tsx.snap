// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GoogleSearchLink should match snapshot 1`] = `
<DocumentFragment>
  <a
    href="https://www.google.com/search?q=roberto"
    rel="noreferrer"
    target="_blank"
  >
    roberto
  </a>
</DocumentFragment>
`;
